lib/mllp/sender.ex

```diff
@@ -236,8 +236,7 @@ defmodule MLLP.Sender do
         message
       end
 
-    state.tcp.send(state.socket, payload)
-    |> case do
+    case state.tcp.send(state.socket, payload) do
```

Comment:
Nit fix :) Can revert if not desired. There's a few of these, but will only comment here.

Reply:
Is this more preferred way of writing case? In our team, we use it all the time. I don't have strong opinion on either approach.

Reply:
As always when it comes to style and "readability" it depends on who are you talking to. I do think this reads better. Perhaps if we were piping a few times it might be worth it, but  when ever I see a single pipe, I see that as an opportunity  to take it out. 

Piping to blocks (in this case, a case expression) is another story, and very bias on that part since  I wrote this https://github.com/rrrene/credo/blob/master/lib/credo/check/readability/block_pipe.ex.

There's a credo check for single pipe  usage that's enabled by default iirc.  

All that said, at the end of the day it's what the team prefers. There's never a right or wrong answer, and terms like  "readability"  are impossible to define. 

I adjusted the above because my I kept having trouble reading it,  and wasn't planning on pushing it up, but it  here is none the less :)